src/Symfony/Component/Messenger/Command/ConsumeMessagesCommand.php

```diff
@@ -86,7 +86,17 @@ protected function configure(): void
      */
     protected function execute(InputInterface $input, OutputInterface $output): void
     {
-        if (!$this->receiverLocator->has($receiverName = $input->getArgument('receiver'))) {
+        $receiverName = $input->getArgument('receiver');
+
+        if (empty($receiverName)) {
+            $receiverName = '.default';
```

Comment:
What about injecting the default receiver name in the constructor instead (optional).
If provided, use it as default value for the `receiver` arg and make it optional. If not, keep it mandatory. When using the component standalone, it'll make sense to define a default this way.
With the framework bundle, do only provide the default if there is a single receiver.

Reply:
What about showing the list of receivers available and selecting one if none were provided? thus we avoid double check on receiver names ;)

Reply:
But that means also injecting the whole list of receiver names, as the `ContainerInterface` does not allow to get defined keys. So, should be optional IMHO.

Reply:
> What about injecting the default receiver name in the constructor instead (optional)?

Yes! Much better idea :)

Reply:
> What about showing the list of receivers available and selecting one

Shall we do that in another PR? I'm not 100% sure it makes sense to be honest so I'd prefer somebody else to work on that 😜

Reply:
Fine for another PR. I'm sure @yceruto would be glad to do it. Otherwise I'll do :)

Reply:
I've updated it based on your reviews, it's much better 😉